php/php.blade/src/org/netbeans/modules/php/blade/editor/actions/AntlrDebugAction.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.php.blade.editor.actions;
+
+import java.awt.event.ActionEvent;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import javax.swing.JComponent;
+import javax.swing.JMenu;
+import javax.swing.JMenuItem;
+import javax.swing.JPopupMenu;
+import javax.swing.event.PopupMenuEvent;
+import javax.swing.event.PopupMenuListener;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionRegistration;
+import org.openide.awt.Actions;
+import org.openide.awt.DynamicMenuContent;
+import org.openide.nodes.Node;
+import org.openide.util.ContextAwareAction;
+import org.openide.util.HelpCtx;
+import org.openide.util.Lookup;
+import org.openide.util.actions.Presenter;
+
+/**
+ *
+ * @author bhaidu
+ */
+@ActionID(id = "org.netbeans.modules.php.blade.editor.actions.AntlrDebug", category = "System")
+@ActionRegistration(displayName = "Antlr Debug", lazy=false)
+public class AntlrDebugAction extends AbstractAction implements ContextAwareAction, Presenter.Popup {
+
+    private Node node;
+
+    public void setNode(Node node){
+        this.node = node;
+    }
+
+    @Override
+    public Action createContextAwareInstance(Lookup lkp) {
+        Node folderNode = lkp.lookup(Node.class);
+        AntlrDebugAction act =  new AntlrDebugAction();
+        act.setNode(folderNode);
+        return act;
+    }
+
+    static final long serialVersionUID = 4906417339959070129L;
```

Comment:
```suggestion
    private static final long serialVersionUID = 4906417339959070129L;
```
should move this before `private  Node node;`